atexit_test.go

```diff
@@ -16,19 +16,28 @@ func TestRegister(t *testing.T) {
 	}
 }
 
+func TestUnregister(t *testing.T) {
+	current := len(handlers)
+	handlerId := Register(func() {})
+	Unregister(handlerId)
+	if len(handlers) != current {
+		t.Fatalf("can't remove handler")
+	}
+}
+
 func TestHandler(t *testing.T) {
 	err := exec.Command("go", "install").Run()
 	if err != nil {
 		t.Fatalf("can't install - %s", err)
 	}
 
-	gofile := "/tmp/atexit-testprog.go"
+	gofile := "./testprog/atexit-testprog.go"
```

Comment:
How is `testprog` better than `/tmp`? Windows? I prefer to use `ioutil.TempDir` in this case

Reply:
That's just because whith /tmp my Travis CI don't want to build. So your free to use what you want. Don't why build pass on my Travis and not your's and vice versa